### DISCUSSION TITLE
使用docker把阿里云盘挂在成功之后在ubuntu使用davfs2挂载报错

### DISCUSSION THREAD

**Discussion by AdoShan** [messense/aliyundrive-webdav#387] [Q&A] [answered]
如题，应该咋解决啊。想挂载到Ubuntu上来。docker本身也是在Ubuntu上跑的

使用命令如下
`mount -t davfs http://localhost:12318 /mnt/WebDav/`

报错如下
`/sbin/mount.davfs: loading kernel module fuse
modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/5.13.19-2-pve/modules.dep.bin'
modprobe: FATAL: Module fuse not found in directory /lib/modules/5.13.19-2-pve
/sbin/mount.davfs: loading kernel module fuse failed
/sbin/mount.davfs: waiting for /dev/fuse to be created
/sbin/mount.davfs: can't open fuse device
/sbin/mount.davfs: trying coda kernel file system
/sbin/mount.davfs: no free coda device to mount`

我搜到说运行`--privileged --cap-add=SYS_ADMIN --device /dev/fuse`不知道咋搞

**Comment by messense** [comment, marked as answer]
非本项目问题，davfs2 问题请自行搜索、排查解决。

**Reply by AdoShan** [reply]
解决了。加两句
`sudo mknod /dev/fuse c 10 229

sudo update-rc.d -f udev disable`
给大家一个解决方案

**Reply by tuofadesanmao** [reply]
> 解决了。加两句 'sudo mknod /dev/fuse c 10 229
> 
> sudo update-rc.d -f udev disable' 给大家一个解决方案

加上这两句确实能解决，但具体原因是因为啥啊  还有其他办法解决不
